src/Microsoft.DotNet.Tools.Repl.Csi/Program.cs

```diff
@@ -19,21 +24,128 @@ public static int Main(string[] args)
             app.FullName = "C# REPL";
             app.Description = "C# REPL for the .NET platform";
             app.HelpOption("-h|--help");
+
             var script = app.Argument("<SCRIPT>", "The .csx file to run. Defaults to interactive mode.");
+            var framework = app.Option("-f|--framework <FRAMEWORK>", "Compile a specific framework", CommandOptionType.MultipleValue);
+            var configuration = app.Option("-c|--configuration <CONFIGURATION>", "Configuration under which to build", CommandOptionType.SingleValue);
+            var preserveTemporary = app.Option("-t|--preserve-temporary", "Keep the output's temporary directory around", CommandOptionType.NoValue);
```

Comment:
@cston why should we preserve the temp dir? It will needlessly keep eating up space on the machine as we will create a new directory every time the repl is seeded.